### PR TITLE
pull my work

### PR DESCRIPTION
**Pull request by yeombang87**
- .gitignore file has been modified to suit various development environments.
- OS support : Windows, macOS
- Development tool : VS Code, WebStorm, SublimeText
- Other : Node, Vue, Python

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,13 +416,18 @@
 *.out*
 *.dat*
 *.report
-__pycache__
```
Comment (iipeace) [inline .gitignore]: isn't needed to ignore python cache files? :)

File: .gitignore [hunk 2]
```diff
@@ -8,13 +416,18 @@
...
 build
 static
+/dist
 
-# virtualenv
-.python-version
```
Comment (iipeace) [inline .gitignore]: isn't needed to ignore? :)

**Comment by iipeace** [comment]
could you check this? :)

**Comment by yeombang87** [comment]
I will check it, fix it, and reflect it again.

**Pull request closed without merging**